### DISCUSSION TITLE
Disable YouTube right click menu

### DISCUSSION THREAD

**Discussion by VoidCruzer** [Araxeus/Youtube-Volume-Scroll#106] [Q&A] [answered]
Would it be possible to disable the right click menu from popping up on YouTube videos when using the right click to scroll volume method? After releasing the right click button when adjusting volume on the video, it brings up the YouTube video menu, which makes you click off of it again to dismiss. Loving the extension otherwise. Thanks!

**Comment by Araxeus** [comment, marked as answer]
Hey @VoidCruzer, what you are describing shouldn't be happening, **what browser are you on?**

I've tested just now on both Chrome and Firefox and releasing the right click after scrolling doesn't open the menu for me.

> [!NOTE]  
> if you know javascript, the code that's supposed to fix this problem is: https://github.com/Araxeus/Youtube-Volume-Scroll/blob/fcea647ffa189450c63bcc8d8ac3876f68da9eac/unpacked/pageAccess.js#L173-L196
